src/guessTheNumber.js

```diff
@@ -0,0 +1,16 @@
+const randomNum = Math.floor(Math.random() * 100)
```

Comment:
Could randomNum been part of the guessTheNumber function. It probably should if it is important to that function and would not affect implementation.